src/crypter.js

```diff
@@ -20,7 +22,7 @@ module.exports.Crypter = (secret: Buffer, {
   gzip = constants.GZIP_MODE.AUTO,
   encryption = 'aes-256-cbc',
   ivLength = 16
-}: {
+}: { // I would move this above to a type alias just to unclutter reading...
```

Comment:
Agreed. Flow is very verbose here. Ideally you could just comment after the variables...

Reply:
I would put a type alias right above this function as `type options = ...`.

Reply:
moved